do not use  bdrv_drain_all()

This seems to hang now (qemu 2.2), causing timeouts.

Index: new/blockdev.c
===================================================================
--- new.orig/blockdev.c	2014-12-29 06:27:15.000000000 +0100
+++ new/blockdev.c	2014-12-29 07:11:53.000000000 +0100
@@ -2107,9 +2107,6 @@
         vma_writer_set_error(backup_state.vmaw, "backup cancelled");
     }
 
-    /* drain all i/o (awake jobs waiting for aio) */
-    bdrv_drain_all();
-
     GList *l = backup_state.di_list;
     while (l) {
         PVEBackupDevInfo *di = (PVEBackupDevInfo *)l->data;
@@ -2118,8 +2115,7 @@
             BlockJob *job = di->bs->job;
             if (job) {
                 if (!di->completed) {
-                     block_job_cancel_sync(job);
-                     bdrv_drain_all(); /* drain all i/o (awake jobs waiting for aio) */
+                    block_job_cancel_sync(job);
                 }
             }
         }
